sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Integration.Tests/FunctionalTests/AzureMonitorTestBase.cs

```diff
@@ -0,0 +1,167 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Integration.Tests.FunctionalTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+
+    using Azure.Core.Pipeline;
+    using Azure.Core.TestFramework;
+    using Azure.Identity;
+
+    using Microsoft.Azure.ApplicationInsights.Query;
+    using Microsoft.Azure.ApplicationInsights.Query.Models;
+    using Microsoft.Rest.Azure.Authentication;
+
+    using NUnit.Framework;
+
+    public abstract class AzureMonitorTestBase : RecordedTestBase<AzureMonitorTestEnvironment>
+    {
+        public int FlushTimeoutMilliseconds = 1000;
```

Comment:
This seems to be very aggressive.

Reply:
What would you recommend here?
I assumed this would be appropriate because our test scenarios are very small. 
This current test generates only a single telemetry item.

Reply:
Normally I've seen people putting 10s, 15s or 1m for cross data call.
I guess putting 15s won't harm (since `ForceFlush` would return right after data got flushed)?